setup/environment-macos-arm64.yml

```diff
@@ -0,0 +1,199 @@
+name: hummingbot
+channels:
+  - conda-forge
+  - defaults
+dependencies:
```

Comment:
Changes in the environment are not related to bittrex task. They should not be included in this PR. Please remove this file.

Reply:
Undid changes to env